### PR TITLE
Added PackagesEnabled field to Project

### PR DESCRIPTION
**Pull request by alxrem**
Save value of project attribute "packages_enabled" to the PackagesEnabled field of Project structure.

### CODE REVIEW

File: projects.go [hunk 1]
```diff
@@ -62,6 +62,7 @@ type Project struct {
 	SnippetsEnabled                           bool               `json:"snippets_enabled"`
 	ResolveOutdatedDiffDiscussions            bool               `json:"resolve_outdated_diff_discussions"`
 	ContainerRegistryEnabled                  bool               `json:"container_registry_enabled"`
+	PackagesEnabled                           bool               `json:"packages_enabled"`
```
Comment (svanharmelen) [inline projects.go:65]: Please place this one between `MirrorOverwritesDivergedBranches` and `ServiceDeskEnabled` so it follows the ordering in the docs. Thanks!

Comment (alxrem) [reply projects.go]: Done in the commit below.

**Review by svanharmelen — APPROVED** [review]
Thanks @alxrem! LGTM!

**Comment by jeremad** [comment]
wow I was about to do the exact same PR. Thanks, looks forward for a release <3

**Pull request merged**